include/aie/Dialect/AIE/IR/AIE.td

```diff
@@ -1319,6 +1321,10 @@ def AIE_ObjectFifoCreateOp: AIE_Op<"objectFifo.createObjectFifo", []> {
     buffers (of given output type) and their locks in the Memory Module of the appropriate tile(s) after lowering, 
     based on tile-adjacency. These elements represent the conceptual depth of the objectFifo.
 
+    For the producer and for each consumer, a different size (i.e., element number) can be specified. This will 
```

Comment:
Could we get the best of both worlds by keeping the simple analysis here and not requiring the sizes to be specified?  What's a case that the analysis gets wrong?

Reply:
Currently, the simple analysis I implemented always looks for the maximum acquire number for an objFifo in a core and then adds +1 to it to account for pre-fetching. Right now, even if a size is specified it may be ignored if it's larger than the number calculated before.

This update specifically concerns broadcasting. Broadcast has multiple consumers but doesn't account for any connections between them, just between the producer and each consumer. So for example, if consumers A and B receive data through broadcast from a producer P using ping pong buffering, but B also needs data from A (calculated based on data from P), higher concurrency may be achieved if instead of a ping pong buffer B had a circular buffer of size 3 (this way, P can push data faster to both consumers).

Reply:
OK, then I would tend to view this as an optimization:  'Most of the time you shouldn't need to specify the size, because the the analysis will compute a minimum safe value.  However it's also possible to specify a larger size, which may increase performance in some cases'.  this leaves open the possibility of a more complex analysis that would take into account the situation you describe.

Reply:
Yes, that makes sense! I'll update the PR.